lib/tasks/scraper.rake

```diff
@@ -13,4 +13,21 @@ namespace :scraper do
       puts "Use FORCE=true to force this task"
     end
   end
+
+  desc "one-off task to extract attachments from bulletins"
+  task back_extract_bulletins: :environment do
+    # Checking for bulletin: false since we want to back extract Bulletins
+    # that existed before we added that flag.
+    StandardsImport.where(bulletin: false).where("notes LIKE ?", "%BulletinsJob%").find_each do |standards_import|
```

Comment:
@littleforest should we add an `includes` to load any additional association and avoid an n+1 query?

Reply:
In the Administrate PR that I just merged, I added [this includes](https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/blob/main/app/models/standards_import.rb#L23). I'm having trouble getting an `includes` to work directly on the StandardsImport class.